.coafile

```diff
@@ -22,7 +22,7 @@ default_actions = *: ApplyPatchAction
 # Do not set default_action to ApplyPatchAction as it may lead to some
 # required imports being removed that might result in coala behaving weirdly.
 
-default_action: *: ShowPatchAction
+default_action = *: ShowPatchAction
```

Comment:
isn't it actually `default_actions`? (note the `s`)

Reply:
@Makman2 it works both ways https://github.com/coala/coala/blob/b5d7973a73badda75ee362e2b3c25df3b0226db6/.coafile#L25

Reply:
although didn't test what would happen if more than one action is given without the `s`